README.md

```diff
@@ -37,6 +38,8 @@ e.g. `wttrbar --date-format "%m/%d" --location Paris --hide-conditions`
 
 To display the weather icons correctly, you will need to have a font that supports emojis installed. The screenshot uses [Noto Emoji](https://github.com/googlefonts/noto-emoji), but you can use [other fonts](https://wiki.archlinux.org/title/fonts#Emoji_and_symbols) too.
 
+With the `--nerd` argument the emojis will be replaced by [nerd font](https://www.nerdfonts.com/) symbols.
+
```

Comment:
No need to add it twice in the README, it's enough to add it at line 27

Reply:
good point